packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -426,6 +427,87 @@ The `EvaluateExpression` supports a `runtime` prop to specify the Lambda
 runtime to use to evaluate the expression. Currently, the only runtime
 supported is `lambda.Runtime.NODEJS_10_X`.
 
+#### DynamoDB example
+
```

Comment:
Break this up into separate code snippet for each type of task, rather than one big one.